stack-7.8.yaml

```diff
@@ -76,6 +76,7 @@ extra-deps:
 - text-metrics-0.1.0
 - pid1-0.1.0.0
 - file-embed-0.0.10
+- yaml-0.8.20
```

Comment:
I expect we'll need this `- yaml-0.8.20` also in `stack.yaml` and `stack-8.0.yaml` since the snapshots use older versions, but CI will tell.